demos/common/include/samples/args_helper.hpp

```diff
@@ -9,11 +9,25 @@
 
 #pragma once
 
+#include <limits>
 #include <map>
 #include <set>
 #include <string>
 #include <vector>
 
+#define DEFINE_INPUT_FLAGS \
+static const char input_message[] = "Required. An input to process. The input must be a single image, a folder of " \
+    "images or anything that cv::VideoCapture can process."; \
+static const char loop_message[] = "Optional. Enable reading the input in a loop."; \
+DEFINE_string(i, "", input_message); \
+DEFINE_bool(loop, false, loop_message);
+
+constexpr uint32_t uint32Size_tMinMax() noexcept {
+    return std::numeric_limits<uint32_t>::max() < std::numeric_limits<size_t>::max()
+        ? std::numeric_limits<uint32_t>::max()
+        : static_cast<uint32_t>(std::numeric_limits<size_t>::max());
+}
```

Comment:
Isn't this basically just `uint32_t(std::numeric_limits<size_t>::max())`?

Reply:
Updated